include/nuraft_mesg/nuraft_mesg.hpp

```diff
@@ -79,7 +79,7 @@ class Manager {
                                   std::shared_ptr< mesg_state_mgr >) = 0;
 
     // Send a client request to the cluster
-    virtual NullAsyncResult add_member(group_id_t const& group_id, peer_id_t const& server_id) = 0;
+    virtual NullAsyncResult add_member(group_id_t const& group_id, nuraft::srv_config const& srv_config) = 0;
```

Comment:
May be add another overload 
```suggestion
    virtual NullAsyncResult add_member(group_id_t const& group_id, peer_id_t const& server_id) = 0;
    virtual NullAsyncResult add_member(group_id_t const& group_id, nuraft::srv_config const& svr_config) = 0;
```

Reply:
Sure, good suggestion.